pkg/controllers/internalmembercluster/v1beta1/member_controller.go

```diff
@@ -56,8 +56,8 @@ const (
 	// EventReasonInternalMemberClusterLeft is the event type and reason string when the agent left.
 	EventReasonInternalMemberClusterLeft = "InternalMemberClusterLeft"
 
-	// we add +-5% jitter
-	jitterPercent = 10
+	// we add +-7.5% jitter
+	jitterPercent = 15
```

Comment:
@zhiying-lin  increased the jitter here, is +-7.5% or do we want a more well-rounded number where it is +-10%?

Reply:
i feel 10% is too much or you can round to 7%, using 14 instead.

Let's start with a small number. You'll know the diff after you add the log.